requirements.txt

```
@@ -1,7 +1,7 @@
 asyncio-mqtt==0.12.1
 bitstring==3.1.9
 black==22.10.0
-git+https://github.com/minims/python-bluetooth-mesh@43f3b368add0e0a40782478369a30454b95a9034
+git+https://github.com/louisjennings/python-bluetooth-mesh@dc4132274b0883c9296eab9dd0998a345280e051
```

```suggestion
git+https://github.com/minims/python-bluetooth-mesh@43f3b368add0e0a40782478369a30454b95a9034
```
I have rebase my fork to latest version of the official repo

---

The HSL functionality depends upon changes I made in the required commit. I'll eventually try getting the HSL models included in the official repo. Until then, I can create a pull request from the feature branch to your repo.